tools/run-tests-zkasm.js

```diff
@@ -21,14 +21,20 @@ async function main() {
     // Get all zkasm files
     const pathZkasm = path.join(process.cwd(), process.argv[2]);
     const files = await getTestFiles(pathZkasm);
-
+    
+    let exit_code = 0;
     // Run all zkasm files
     // eslint-disable-next-line no-restricted-syntax
     console.log(chalk.yellow('--> Start running zkasm files'));
     for (const file of files) {
         if (file.includes('ignore'))
             continue;
-        await runTest(file, cmPols);
+        if (await runTest(file, cmPols) == 1) {
+            exit_code = 1;
```

Comment:
Given that we don't really care about the exact exit code and we don't use it below, maybe we can change the meaning of this variable to make the code easier to read. E.g. we can name it `failed_test_count` and report, in the end, the number of failed tests or we can just track a boolean value `have_failed_test`.